mumbleBot.py

```diff
@@ -314,6 +318,17 @@ def is_admin(self, user):
         else:
             return False
 
+    # =======================
+    #         Users changed
+    # =======================
+
+    def users_changed(self, user, message):
+        own_channel = self.mumble.channels[self.mumble.users.myself['channel_id']]
+        # if the bot is the only user left in the channel
+        if len(own_channel.get_users()) == 1: 
+            self.log.info('Other users in the channel left. Stopping music now.')
```

Comment:
I'd suggest changing this line into `self.log.info('bot: Other users in the channel left. Stopping music now.')` , just to follow the logging convention of this project. :)

Reply:
Thank you, fixed it. (: